bin/dotd

```diff
@@ -22,7 +22,8 @@ class String
   end
 end
 
-LOG_FILE = '/var/log/code-dot-org-dotd.log'
+BIN_DIR = File.expand_path(File.dirname(__FILE__))
+LOG_FILE = "#{BIN_DIR}/log/dotd.log"
```

Comment:
Does this "work" if the user has made a local copy of the script? IIRC, this practice was encouraged, as the script is (intentionally) dependency free.